proto/babylon/btcstaking/v1/events.proto

```diff
@@ -215,8 +215,10 @@ message EventBTCDelgationUnbondedEarly {
   // staking_tx_hash is the hash of the staking tx.
   // It uniquely identifies a BTC delegation
   string staking_tx_hash = 1 [(amino.dont_omitempty) = true];
+  // start_height is the start BTC height of the early unbonding
+  string start_height = 2 [(amino.dont_omitempty) = true];
```

Comment:
Why is this not a uint32 or uint64? Just curious.

Reply:
This is for easier parsing for subscribers of the event